config/initializers/active_support_extensions.rb

```diff
@@ -0,0 +1,30 @@
+#this makes sure that hash with indifferent access gets deep converted into hash.
```

Comment:
literally LOL
like, people from the other side of the office are looking at me funny. I
think I broke something with the sound of my voice.